student-store-ui/src/components/App/App.jsx

```diff
@@ -6,14 +6,19 @@ import Home from "../Home/Home"
 import "./App.css"
 
 export default function App() {
+
+  // Global States
+  const url = 'http://localhost:3001'
+  const [itemList, itemListSet] = React.useState(null)
+  const [productCart, productCartSet] = React.useState(new Map())
+
```

Comment:
omit importing react and use `useState()` directly (instead of `React.useState()`)